core/src/main/java/org/elasticsearch/action/get/GetResponse.java

```diff
@@ -168,4 +169,17 @@ public void writeTo(StreamOutput out) throws IOException {
         super.writeTo(out);
         getResult.writeTo(out);
     }
+
+	@Override
+	public String toString() {
+		try {
```

Comment:
@nik9000 It's exactly the same as `SearchResponse.toString()`: https://github.com/elastic/elasticsearch/blob/master/core/src/main/java/org/elasticsearch/action/search/SearchResponse.java#L233-L244

And if you meant `org.elasticsearch.common.Strings.toString(ToXContent toXContent, boolean wrapInObject)`: https://github.com/elastic/elasticsearch/blob/master/core/src/main/java/org/elasticsearch/common/Strings.java#L995-L1009

it's almost same except pretty-printing which has a value as a log message.